.dockerignore

```diff
@@ -27,7 +27,7 @@ docs
 
 # Assets
 data
-src/frontend
+src/frontend/node_modules
```

Comment:
I would suggest you to use `**/node_modules` instead.

Reply:
Good idea :+1:

Reply:
done